src/Pim/Bundle/CatalogBundle/tests/integration/Product/RemoveOptionalAttributeIntegration.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\Product;
+
+use Akeneo\Test\Integration\Configuration;
+use Akeneo\Test\Integration\TestCase;
+use PHPUnit\Framework\Assert;
+use Pim\Bundle\CatalogBundle\tests\fixture\EntityBuilder;
+use Pim\Component\Catalog\Model\ProductInterface;
+
+class RemoveOptionalAttributeIntegration extends TestCase
+{
+    /**
+     * @test
+     */
+    function remove_an_attribute_on_a_product_without_family()
```

Comment:
```suggestion
    function test_remove_an_attribute_on_a_product_without_family()
```
Otherwise it won' be run by the CI